### PR TITLE
Updated log client to read environment variables with sane defaults.

### PR DESCRIPTION
**Pull request by nicksantamaria**
This fixes an issue with application logs not being aggregated on the new Azure cluster.

### CODE REVIEW

File: packages/ripple-nuxt-tide/lib/core/logger.js [hunk 1]
```diff
@@ -53,19 +53,17 @@ const errorPrint = format(info => {
...
   const gitBranch = process.env.LAGOON_GIT_SAFE_BRANCH || LAGOON_LOGS_DEFAULT_SAFE_BRANCH
-  const type = [openshiftProject, gitBranch]
+  const type = [k8sNamespace, gitBranch]
   info.type = type.join('-')
   return info
 })
 
 // Set up level based on env settings.
-let logLevel = 'info'
+let logLevel = process.env.LOG_LEVEL || 'warn'
```
Comment (tim-yao) [inline packages/ripple-nuxt-tide/lib/core/logger.js:64]: 👍

**Review by tim-yao — APPROVED** [review]
Thanks @nicksantamaria , looks good to me.

**Comment by nicksantamaria** [comment]
@tim-yao I was looking at the possibility of refactoring how the log transports were added, so the console was only used for local dev + clientside. But never got it working. 


Lets go with this PR as-is. Merge if you're happy with it.

**Comment by tim-yao** [comment]
> @tim-yao I was looking at the possibility of refactoring how the log transports were added, so the console was only used for local dev + clientside. But never got it working.
> 
> Lets go with this PR as-is. Merge if you're happy with it.



> @tim-yao I was looking at the possibility of refactoring how the log transports were added, so the console was only used for local dev + clientside. But never got it working.
> 
> Lets go with this PR as-is. Merge if you're happy with it.

@nicksantamaria Thanks. I will have a quick look to see if I can disable console log for lagoon in this PR. If not I will merge it.

**Pull request merged**